holoviews/plotting/bokeh/callbacks.py

```diff
@@ -547,14 +567,177 @@ def _process_msg(self, msg):
         return self._transform(dict(msg, stroke_count=self.stroke_count))
 
 
-class TapCallback(PointerXYCallback):
+class PopupMixin:
+
+    geom_type = 'any'
+
+    def initialize(self, plot_id=None):
+        super().initialize(plot_id=plot_id)
+        if not self.streams:
+            return
+
+        self._existing_popup = None
+        stream = self.streams[0]
+        if not getattr(stream, 'popup', None):
+            return
+        elif Panel is None:
+            raise VersionError("Popup requires Bokeh >= 3.4")
+
+        close_button = Button(label="", stylesheets=[r"""
+        :host(.bk-Button) {
+            width: 100%;
+            height: 100%;
+            top: -1em;
+        }
+        .bk-btn, .bk-btn:hover, .bk-btn:active, .bk-btn:focus {
+            background: none;
+            border: none;
+            color: inherit;
+            cursor: pointer;
+            padding: 0.5em;  # increase hit box or else playwright can't click
```

Comment:
Can you try [evaluate](https://stackoverflow.com/questions/70361865/how-to-get-the-locator-for-an-css-after-selector) in the test (if you haven't already)?

Adding a Python comment in CSS also seems a bit weird.